### PR TITLE
UDP delay test fix

### PR DESCRIPTION
**Pull request by ashamis**

### CODE REVIEW

File: tests/infra/libbyz/node.py [hunk 1]
```diff
@@ -55,7 +55,11 @@ def stop(self):
         logger.info(f"IP: {self.machine_name} - terminating process")
         if self.proc:
             self.proc.terminate()
-            self.proc.wait()
+            result = self.proc.wait(5)
+            if result != 0:
+                self.proc.kill()
+                self.proc.wait()
+
```
Comment (olgavrou) [inline tests/infra/libbyz/node.py]: I was thinking maybe we could `cat` the `out\<port\>.txt` files when we know something isn't right. We can do that here and if the `syscall_stats` assertion is going to fail.

**Comment by achamayou** [comment]
Another occurrence: https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=4131&view=logs&j=153a2874-5544-596c-fb23-dd6c43726782&t=ef2a4157-9405-5cf5-3348-6460c898fe08

**Comment by cimetrics** [comment]
udp_delay@[4168 aka 20200128.29](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4168&view=results) vs master ewma over 20 builds from [3907](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3907&view=results) to [4161](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4161&view=results)
images

File: tests/infra/libbyz/node.py (hunk 1 above)
Comment (ashamis) [reply tests/infra/libbyz/node.py]: this is a bigger work item #734

**Pull request merged**